data/specials/hole.j3x

```diff
@@ -0,0 +1,19 @@
+string default_activation="OnEnter"
+list default_functions=["Teleport"]
+list default_datas=[(0,0)]
+
+vector size=(48.0,37.0)
+vector collision_offset=(0.0,-18.0)
+vector map_offset=(0.0,0.0)
+float collision_radius=3.0
```

Comment:
Previous collision_size was risky - 1.0 height is only 1 pixel and there can be a lot of computational errors due to floating point precision. E.g. when FPS are less than 30, everything (including collisions) is computed with longer time steps. It means that between two steps the player position differs more - the player can even "jump" between frames and avoid collision check with small collision objects. I changed it to collision_radius = 3.0.